cucumber/authenticators_jwt/features/authn_jwt_audience.feature

```diff
@@ -0,0 +1,68 @@
+Feature: JWT Authenticator - Check audience
+
+  Verify the authenticator works correctly audience variable and aud claim
+
+  Background:
```

Comment:
Can these test be moved to authn_jwt_check_standard_claims.feature?

Reply:
Agreed